test/util/edu-helpers.js

```diff
@@ -16,6 +18,35 @@ function completeVeteranInformation(client, data, onlyRequiredFields) {
   }
 }
 
+function completeRelativeInformation(client, data, onlyRequiredFields) {
+  const dobFields = data.relativeDateOfBirth.split('-');
+  client
+    .clearValue('input[name="root_relativeFullName_first"]')
+    .setValue('input[name="root_relativeFullName_first"]', data.relativeFullName.first)
```

Comment:
I like this idea, though we'll have to stop chaining them as client methods, e.g. will look something like
```
clearAndSetValue(client, 'input[name="root_relativeFullName_first"]', data.relativeFullName.first);
clearAndSetValue(client, 'input[name="root_relativeFullName_last"]', data.relativeFullName.last);
```
Definitely prefer to do this in a separate PR though, since it's been a struggle to get these working. (So much flakiness locally, and then different results when running locally vs. on jenkins 😭 )